test/test-remote/looker/src/index.ts

```diff
@@ -105,22 +103,28 @@ async function main() {
   parseCmdlineArgs();
   const httpServerTerminator = pm.setupPromExporter();
 
-  //let dummystreams: Array<LogSeries>;
   let dummystreams: Array<LogSeries | MetricSeries>;
 
   for (let cyclenum = 1; cyclenum < CFG.n_cycles + 1; cyclenum++) {
     setUptimeGauge();
 
     const numString = `${cyclenum.toString().padStart(3, "0")}`;
-    const guniqueCycleId = `${CFG.invocation_id}-${numString}-${rndstring(4)}`;
+    // invocation_id contains 10 characters of random data by default
+    const guniqueCycleId = `${CFG.invocation_id}-${numString}`;
```

Comment:
we need to make sure there is no (realistic) upper bound for the number of cycles we can run -- I think that is why I had `${rndstring(4)}`. Maybe change `padStart(3, "0")}` to `padStart(X, "0")}` with X being larger than 3 :).

Also.. what do you think: I somehow like fixed width IDs but maybe here we don't want one after all. Just use `cyclenum`, w/o zero-padding?

Reply:
dropping padding is fine

Reply:
dropping padding in followup PR